stdlib/uuid.pyi

```diff
@@ -67,6 +68,11 @@ class UUID:
 def getnode() -> int: ...
 def uuid1(node: int | None = None, clock_seq: int | None = None) -> UUID: ...
 
+if sys.version_info >= (3, 14):
+    def uuid6(node: int | None = None, clock_seq: int | None = None) -> UUID: ...
```

Comment:
nit but it would have been nice to have these in order (after uuid5)

Reply:
Thanks, probably not worth the trouble! Maybe the next time somebody touches this module